app/src/main/java/com/codeforireland/vacanthome/viewmodels/PhotoViewModel.java

```diff
@@ -0,0 +1,29 @@
+package com.codeforireland.vacanthome.viewmodels;
+
+import android.app.Activity;
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.ViewModel;
+import android.graphics.Bitmap;
+
+
+public class PhotoViewModel  extends ViewModel{
+
+    private final static String TAG = PhotoViewModel.class.getSimpleName();
+
+    private MutableLiveData<Bitmap> mutableLiveData;
+
+    public LiveData<Bitmap> getPicture(Activity activity){//or just pass intent???
```

Comment:
Can you explain? Can we just pass the intent? Is there an advantage to doing it this way?

Reply:
I was considering whether to use camera API or  invoke an Intent.